roles/ci_gen_kustomize_values/templates/ovs-dpdk-sriov/edpm-values/values.yaml.j2

```diff
@@ -0,0 +1,74 @@
+  baremetalSetTemplate:
+    ctlplaneInterface: {{ cifmw_networking_env_definition.instances[instance].networks.ctlplane.interface_name }}
+    cloudUserName: cloud-admin
+    provisioningInterface: {{ cifmw_ci_gen_kustomize_values_provisioning_interface }}
+    bmhLabelSelector:
+      app: openstack
+    passwordSecret:
+      name: baremetalset-password-secret
+      namespace: openstack
+  ssh_keys:
+    authorized: {{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | b64encode }}
+    private: {{ cifmw_ci_gen_kustomize_values_ssh_private_key | b64encode }}
+    public: {{ cifmw_ci_gen_kustomize_values_ssh_public_key | b64encode }}
+  nova:
+    migration:
+      ssh_keys:
+        private: {{ cifmw_ci_gen_kustomize_values_migration_priv_key | b64encode }}
+        public: {{ cifmw_ci_gen_kustomize_values_migration_pub_key | b64encode }}
+    pci:
+      conf: |
+        {{ cifmw_ci_gen_kustomize_values_sriov_pci_device_spec }}
+  nodeset:
+    ansible:
+      ansibleUser: "zuul"
+      ansibleVars:
+        edpm_fips_mode: "{{ 'enabled' if cifmw_fips_enabled|default(false)|bool else 'check' }}"
+        timesync_ntp_servers:
+          - hostname: "{{ cifmw_ci_gen_kustomize_values_ntp_srv | default('pool.ntp.org') }}"
+        edpm_network_config_os_net_config_mappings: {{ cifmw_ci_gen_kustomize_values_os_net_configs_mapping }}
+        edpm_network_config_template: {{ cifmw_ci_gen_kustomize_values_network_config_template }}
+        edpm_kernel_args: "{{ cifmw_ci_gen_kustomize_values_kernel_args }}"
+        edpm_tuned_profile: "{{ cifmw_ci_gen_kustomize_values_tuned_profile }}"
+        edpm_tuned_isolated_cores: "{{ cifmw_ci_gen_kustomize_values_isolated_cores }}"
+        edpm_nova_libvirt_qemu_group: "{{ cifmw_ci_gen_kustomize_values_libvirt_qemo_group }}"
+        edpm_ovs_dpdk_pmd_core_list: "{{ cifmw_ci_gen_kustomize_values_ovs_pmd_core_list }}"
+        edpm_ovs_dpdk_socket_memory: "{{ cifmw_ci_gen_kustomize_values_ovs_dpdk_socket_memory }}"
+        edpm_ovs_dpdk_memory_channels: "{{ cifmw_ci_gen_kustomize_values_ovs_dpdk_memory_channels }}"
+        edpm_ovs_dpdk_vhost_postcopy_support: "{{ cifmw_ci_gen_kustomize_values_ovs_dpdk_vhost_postcopy_support }}"
+        edpm_ovn_bridge_mappings: "{{ cifmw_ci_gen_kustomize_values_ovn_bridge_mapping }}"
+        edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings: "{{ cifmw_ci_gen_kustomize_values_sriov_physical_device_mapping }}"
+{% if cifmw_ci_gen_kustomize_values_sshd_ranges | default([]) | length > 0             %}
+        edpm_sshd_allowed_ranges:
+{%   for range in cifmw_ci_gen_kustomize_values_sshd_ranges                            %}
+          - "{{ range }}"
+{%   endfor                                                                            %}
+{% endif                                                                               %}
+    nodes:
+{% for instance in instances_names                                                     %}
```

Comment:
This list is empty in NFV cases, as the `cifmw_networking_env_definition` doesn't have computes.
The main source of information for this type of VAs is the `cifmw_baremetal_hosts` variable. Iterate over that dict to generate this. At the same time, if we are going to use OSP BMO and the IPAM integration, all the keys here but hostName are not needed, keep just that one.

Reply:
> This list is empty in NFV cases, as the `cifmw_networking_env_definition` doesn't have computes. The main source of information for this type of VAs is the `cifmw_baremetal_hosts` variable. Iterate over that dict to generate this. At the same time, if we are going to use OSP BMO and the IPAM integration, all the keys here but hostName are not needed, keep just that one.

Ack